photon-api/src/main/scala/com/linkedin/photon/ml/optimization/DistributedOptimizationProblem.scala

```diff
@@ -87,10 +89,7 @@ protected[ml] class DistributedOptimizationProblem[Objective <: DistributedObjec
 
     val result = (objectiveFunction, varianceComputation) match {
```

Comment:
If this is where you are updating the variance, this is another point of difference

Reply:
Yes. photon-ml will output Hessian diagonal. To output variance, I only need to invert it.  This is for fixed effect model variance update.